README.md

```diff
@@ -4,26 +4,20 @@ ZeroNet Music Streaming Site - http://127.0.0.1:43110/ZeroLSTN.bit
 
 ## Setting up
 
-Install node and npm.
-
-Install gulp.
-
-```
-# npm install -g gulp-cli
-```
+Install yarn: https://yarnpkg.com/lang/en/docs/install/
 
 In the project's root folder, install all needed dependencies.
 
 ```
-$ npm i
+$ yarn install
```

Comment:
If you're looking for a smaller command, `yarn` is the same as `yarn install`.

Reply:
Oh no way, that's cool. Will edit, thanks!